frontend/src/components/polls/polls_table/PollEditModalAdminForm.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useContext } from "react";
+import { Poll } from "../../../models/polls";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import Modal from "react-bootstrap/Modal";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import * as Yup from "yup";
+import { Formik, Form, useFormikContext } from "formik";
+import { TextFormGroup } from "../../utils/forms/TextFormGroup";
+import { SelectGroup } from "../../utils/forms/SelectGroup";
+import { DatePickerField } from "../../utils/forms/DatePickerField";
+import { dateFormatter } from "../../../utils/format";
+import { api } from "../../../services/apiService";
+import { ToastContext, ToastLevel } from "../../../utils/Toast";
+
+export const PollEditModalAdminForm = ({
+    poll,
+    refetch,
+    handleClose
+}: {
+    poll: Poll;
+    refetch: any;
+    handleClose: () => void;
+}) => {
+    const newToast = useContext(ToastContext);
+
+    function StateField() {
+        let items = new Map();
+
+        items.set(
+            "ACCEPTED",
+            <span className="selectgroup-button selectgroup-button-icon">
+                <i className="fe fe-check text-success" /> Accepté
+            </span>
+        );
+
+        items.set(
+            "REJECTED",
+            <span className="selectgroup-button selectgroup-button-icon">
+                <i className="fe fe-x text-danger" /> Refusé
+            </span>
+        );
+
+        items.set(
+            "REVIEWING",
+            <span className="selectgroup-button selectgroup-button-icon">
+                <i className="fe fe-eye text-warning" /> En attente
+            </span>
+        );
+
+        return (
+            <SelectGroup
+                type="pills"
+                label="Statut"
+                name="state"
+                items={items}
+            />
+        );
+    }
+
+    function AdminCommentField() {
```

Comment:
Move as independent component outside of `PollEditModalAdminForm`, but not exported